rfcs/text/refactoring-date-component.md

```diff
@@ -94,12 +96,12 @@ The date passed to the picker component can be updated through a `useEffect` hoo
     formatISOString
   } from "./__internal__/date-utils"
 
-  const DateInput = ({ value, onChange, onBlur, onFocus, inputProps, ...rest }) => {
+  const DateInput = ({ value, onChange, onBlur, onFocus, inputProps, picker, ...rest }) => {
```

Comment:
The convention is typically `renderPicker`

Reply:
Yeah good point I'll update it to that